diesel/src/types/mod.rs

```diff
@@ -1,720 +1,109 @@
-//! Types which represent a SQL data type.
-//!
-//! The structs in this module are *only* used as markers to represent a SQL type.
-//! They should never be used in your structs.
-//! If you'd like to know the rust types which can be used for a given SQL type,
-//! see the documentation for that SQL type.
-//! Additional types may be provided by other crates.
-//!
-//! To see which SQL type can be used with a given Rust type,
-//! see the "Implementors" section of [`FromSql`].
-//!
-//! [`FromSql`]: trait.FromSql.html
-//!
-//! Any backend specific types are re-exported through this module
+// All items in this module are deprecated. They are rendered in docs.
+#![allow(missing_docs)]
+#![cfg(feature = "with-deprecated")]
+#![deprecated(since = "1.1.0", note = "Use `sql_types`, `serialize`, or `deserialize` instead")]
 
-pub mod ops;
-mod ord;
-#[macro_use]
-#[doc(hidden)]
-pub mod impls;
-mod fold;
+#[deprecated(since = "1.1.0", note = "Use `sql_types` instead")]
```

Comment:
The message is not 100% correct, the user should use the types inside the `sql_types` module instead the ones inside the `types` module. If I intentionally try to misinterpret the message I read something like: Instead of `types::Integer` use simply `sql_types`"